src/main/java/org/javarosa/core/model/FormDef.java

```diff
@@ -1101,9 +1098,14 @@ private void evaluateTriggerables(List<Triggerable> tv,
      * @param triggerable The triggerable to be updated
      * @param anchorRef   The reference to the value which was changed.
      */
+    @Trace
```

Comment:
are these trace annotations going to be OK on mobile?

Reply:
I believe so - https://github.com/dimagi/commcare-core/pull/1063#issuecomment-1011491637 , I have not faced any issues so far while testing mobile with this branch.